source/NVDAObjects/__init__.py

```diff
@@ -308,6 +308,17 @@ def findOverlayClasses(self, clsList: typing.List[typing.Type["NVDAObject"]]) ->
 
 	beTransparentToMouse=False #:If true then NVDA will never consider the mouse to be on this object, rather it will be on an ancestor.
 
+	def objectFromPointRedirect(self, x, y):
```

Comment:
Please use type hints:
```suggestion
	def objectFromPointRedirect(self, x: int, y: int) -> NVDAObject | None:
```